packages/create-waku/src/index.ts

```diff
@@ -161,7 +160,9 @@ Options:
 }
 
 async function notifyUpdate() {
-  const packageJson = JSON.parse(readFileSync('../package.json', 'utf8'));
+  const packageJson = JSON.parse(
+    readFileSync(path.resolve(import.meta.dirname, '../package.json'), 'utf8'),
```

Comment:
`import.meta.dirname` isn't available in Node 18.

https://nodejs.org/docs/latest-v18.x/api/esm.html#importmetaurl

Can you try this?
```js
import { readFileSync } from 'node:fs';
const buffer = readFileSync(new URL('./data.proto', import.meta.url));
```

Reply:
yep, that works, thanks!